src/spectral_transform.jl

```diff
@@ -182,8 +181,6 @@ function SpectralTransform( ::Type{NF},                     # Number format NF
 
     # EIGENVALUES (on unit sphere, hence 1/radius²-scaling is omitted)
     eigenvalues = [-l*(l+1) for l in 0:lmax+1]
-    eigenvalues⁻¹ = inv.(eigenvalues)
-    eigenvalues⁻¹[1] = 0                    # set the integration constant to 0
```

Comment:
I just realised that we shouldn't replace `eigenvalues⁻¹` by `1/eigenvalues` as that makes the integration constant infinity. We currently don't test for this (and it's not used by the dynamical core atm) hence the tests still pass
```julia
julia> alms = LowerTriangularMatrix(randn(ComplexF32,33,32));
julia> alms[:,1] .= real.(alms[:,1]);
julia> ∇⁻²alms = copy(alms)
julia> S = SpectralTransform(alms);
julia> SpeedyWeather.∇²!(∇⁻²alms,alms,S,inverse=true)
33×32 LowerTriangularMatrix{ComplexF32}:
   -Inf+NaN*im                0.0+0.0im          …          0.0+0.0im
    -0.369622-0.0im      0.199208-0.408254im                0.0+0.0im
...
```